app/models/notifications/thirteen/thirteen_hearing_scheduled_dnd_staff.rb

```diff
@@ -0,0 +1,27 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/boston-cas/blob/production/LICENSE.md
+###
+
+module Notifications::Thirteen
+  class ThirteenHearingScheduledDndStaff < ::Notifications::Base
+    def self.create_for_match! match
+      match.dnd_staff_contacts.each do |contact|
+        create! match: match, recipient: contact
+      end
+    end
+
+    def decision
+      match.thirteen_hearing_scheduled_decision
+    end
+
+    def event_label
+      "#{Translation.translate('CoC Thirteen')} notified of pending CORI review."
```

Comment:
Just catching this here, but I probably missed it above.  We should probably translate `CORI` in case they want to that to simply read **Hearing** or **CORI/SORI** or something else altogether.
```suggestion
      match.thirteen_hearing_scheduled_decision
    end

    def event_label
      "#{Translation.translate('CoC Thirteen')} notified of pending #{Translation.translate('CORI')} review."
```